server/app/controllers/applicant/ApplicantProgramReviewController.java

```diff
@@ -182,6 +184,25 @@ public CompletionStage<Result> review(Request request, long applicantId, long pr
             });
   }
 
+  @Secure
+  public CompletionStage<Result> review(Request request, long programId) {
+    if (!settingsManifest.getNewApplicantUrlSchemaEnabled()) {
+      // This route is only operative for the new URL schema, so send the user home.
+      return CompletableFuture.completedFuture(redirectToHome());
```

Comment:
question (non-blocking): To check my understanding: `ApplicantRoutes#review` is responsible for not calling this route if the flag is disabled right?

Reply:
That is correct. The defensive code is present in case someone who does not yet have the applicant id in their profile visits this URL directly (got it from someone else, for example).